packages/ubuntu_wizard/test/rounded_list_view_test.dart

```diff
@@ -28,15 +28,15 @@ void main() {
       findsOneWidget,
     );
 
-    // listview inside scrollbar
+    // listview inside rounded container
```

Comment:
It's not visible in the diff, but there's a "scrollbar inside rounded container" test above. Since the internal ListView structure is different than what we had, I changed the "container -> scrollbar -> listview" test to "container -> scrollbar & container -> listview")